app/javascript/react/components/NewReviewFormTile.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+const NewReviewFormTile = (props) => {
+    const newReview = props.newReview
```

Comment:
state for the form should live in the form

when we need to destructure props, we can do so:
```js
const { newReview } = props
```
that way for any props we have we can destructure and list them all
```js
const { newReview, cats, helperMethod } = props
```